README.md

```diff
@@ -1,33 +1,25 @@
-# Document AI Bouding Boxes Workshop 
+# Document AI Notebooks 
 
-In this tutorial you will process a document with a Document AI processor and
-visualize its output.
+This repository contains several [Google AI Platform notebooks][notebooks] 
+to be used with the Cloud [Document AI Platform][docai].
 
-![boxes](images/SampleOutput.png)
-![table](images/SampleFormFields.png)
+![boxes](resources/screenshots/SampleFormFields.png)
 
 ## Prerequisites 
 
-You must be familar GCP and have your own project created.
+You must be familar GCP and have your own project with billing enabled.
 
-1. Set up your GCP project for Document AI following the [Setup Guide](https://cloud.google.com/document-ai/docs/setup)
-1. Enable the 'Document AI API' in your project in the [Cloud Console](https://console.cloud.google.com/ai/document-ai)
-1. Create a Python 3 AI Platform Notebook in your GCP project and clone this repository.
+1. Set up your GCP project for Document AI following the [Setup Guide][set_up].
+1. Enable the 'Document AI API' in your project in the Document AI [Platform][platform].
+1. Create a Python 3 notebook in your GCP project and clone this repository.
```

Comment:
Can we say something like "[Create](https://cloud.google.com/ai-platform/notebooks/docs/create-new) or use an existing instance of AI Platform Notebook with Python 3 using all default configurations." We can also say that Colab or any other Jupyter notebook would work but additional authentication setup may be required.